cmd/airflow.go

```diff
@@ -150,6 +151,7 @@ func newAirflowInitCmd() *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&projectName, "name", "n", "", "Name of Astro project")
 	cmd.Flags().StringVarP(&airflowVersion, "airflow-version", "a", "", "Version of Airflow you want to create an Astro project with. If not specified, latest is assumed. You can change this version in your Dockerfile at any time.")
+	cmd.Flags().StringVarP(&template, "template", "t", "", "Tempate name that you want to use to create the local astro project. Posisble values can be etl, dbt-on-astro, generative-ai and learning-airflow  ")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&template, "template", "t", "", "Tempate name that you want to use to create the local astro project. Possible values can be etl, dbt-on-astro, generative-ai, and learning-airflow  ")
```
Do you use any particular formatting to indicate something can be used as a possible value? we use code formatting or `" " ` to indicate strings in docs and code examples.